content/en/docs/Endpoints/getvideos.md

```diff
@@ -29,7 +29,39 @@ A [`subsonic-response`](../../responses/subsonic-response) element with a nested
 // TODO
 {{< /tab >}}
 {{< tab header="Subsonic" lang="json" >}}
-// TODO
+{
+   "subsonic-response" : {
+      "status" : "ok",
+      "version" : "1.16.1",
+      "videos" : {
+         "video" : [ {
```

Comment:
I think everywhere else that has an array of objects in the markdown JSON is in the form 
```json
[
  {}
]
```
(Specifically, indented an extra level)